.gitignore

```diff
@@ -4,3 +4,5 @@ vendor/
 *.cache
 infection-log.*
 build/
+.idea/
```

Comment:
Please remove IDE specific paths
These should be ignored globally in `~/.gitignore`